app/editor/package.json

```diff
@@ -39,7 +39,7 @@
     "react-tooltip": "4.2.21",
     "redux-logger": "3.0.6",
     "styled-components": "5.3.5",
-    "tno-core": "0.0.165",
+    "tno-core": "0.0.172",
```

Comment:
That's a big jump lol.  You can use the `make link` command to test locally.

Reply:
Changed my mind on the tab colour a few times 😝